csharp/src/Apache.Arrow/Arrays/StringArray.cs

```diff
@@ -35,17 +36,29 @@ public StringArray(ArrayData data)
 
         public override void Accept(IArrowArrayVisitor visitor) => Accept(this, visitor);
 
+        new public ReadOnlySpan<char> Values => ValueBuffer.Span.CastTo<char>();
+        new public ReadOnlySpan<char> GetBytes(int index)
+        {
+            var offset = GetValueOffset(index);
+            var length = GetValueLength(index);
+
+            return Values.Slice(offset, length);
+        }
+
         public string GetString(int index, Encoding encoding = default)
         {
-            encoding = encoding ?? Encoding.UTF8;
+            // I don't think I need encoding here? Whatever chars went in should come out that same?
```

Comment:
Same as the comment. I don't need an encoding anymore because we're no longer manipulating bytes explicitly? Span is doing that for me